data/fields/house.json

```
@@ -1,5 +1,38 @@
 {
     "key": "house",
     "type": "combo",
-    "label": "House Type"
+    "label": "House Type",
+    "strings": {
+        "options": {
+            "bungalow": {
+                "title": "Bungalow",
+                "description": "A small house with a single storey"
+            },
+            "detached": {
+                "title": "Detached House",
+                "description": "A house that is not connected to any other houses"
+            },
+            "link-detached": {
+                "title": "Link-Detached House",
+                "description": "A house that is connected to another house by a garage or other non-living area"
+            },
+            "maisonette": {
+                "title": "Maisonette",
+                "description": "A residential dwelling, split over two floors with private external entrances rather than a communal entrance"
+            },
+            "semi-detached": {
+                "title": "Semi-Detached House",
+                "description": "A house that is attached to another house on one side"
+            },
+            "terrace": {
+                "title": "Row of Terraced Houses",
+                "description": "A row of houses that are attached to each other in a line"
```

```suggestion
                "description": "A row of houses that are attached to each other in a line (mapping each house as a separate area is preferred)"
```

maybe this hint would be useful?